support/src/test/java/com/yubico/yubikit/support/Util.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2022 Yubico.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.yubico.yubikit.support;
+
+import com.yubico.yubikit.core.Transport;
+import com.yubico.yubikit.core.Version;
+import com.yubico.yubikit.management.DeviceConfig;
+import com.yubico.yubikit.management.DeviceInfo;
+import com.yubico.yubikit.management.FormFactor;
+
+import org.jetbrains.annotations.Nullable;
```

Comment:
Is this the correct Nullable annotation? I believe we use `javax.annotation.Nullable` everywher else.

Reply:
fixed